src/test/java/de/qabel/desktop/cellValueFactory/BoxObjectCellValueFactoryTest.java

```diff
@@ -92,11 +92,11 @@ public void testEmpty() {
     }
 
     @Test
-    public void testSize() {
+    public void testFormattedFileSize() {
 
         column = getColumnValue("size", rootNodeFile);
         BoxFile subnode = (BoxFile) rootNodeFile.getChildren().get(0).getValue();
-        assertThat(column.getValue(), is(subnode.getSize().toString()));
+        assertThat(column.getValue(), is(FileUtils.byteCountToDisplaySize(subnode.getSize())));
```

Comment:
checking that `FileUtils.byteCountToDisplaySize(subnode.getSize())` = `FileUtils.byteCountToDisplaySize(((BoxFile)bf).getSize())` is trivial if we know that subnode == bf

You should better set a fixed size (or know the size) and check for the expected value like `assertThat(column.getValue(), is(equalTo("1 KB"));`. That way, the test fails if the output is wrong